app/views/identity_providers/_form.html.erb

```diff
@@ -1,6 +1,6 @@
-<div class="panel panel-default">
-  <%= simple_form_for(@identity_provider) do |f| %>
-    <div class="panel-body">
+<%= simple_form_for(@identity_provider) do |f| %>
```

Comment:
curious about moving the form outside the card; probably okay.

Reply:
Tried both ways and looks the same to my eyes, this seems less awkward this way.

Ex.
![image](https://github.com/samvera/hyku/assets/19597776/9c2804cb-c07c-48a3-975e-6373d57e19e8)